### PR TITLE
Fix the screen logging and keep the most recent logs for each screen

### PR DESCRIPTION
**Pull request by schnaubelt**

### CODE REVIEW

**Comment by astumpf** [comment]
Please do also consider to apply these changes in https://github.com/tu-darmstadt-ros-pkg/workspace_scripts/blob/kinetic-devel/scripts/helper/run_all.sh. I'm wondering if the corresponding lines in run_all.sh are even required?
Finally, please consider to merge these changes also into the more recent noetic branch.

**Comment by schnaubelt** [comment]
The logging behavior in run_all.sh now also has the fixes. As soon as this is approved, I will also add PRs for melodic-devel and noetic-devel

File: scripts/helper/run_all.sh [hunk 1]
```diff
@@ -156,13 +142,19 @@ declare -a directories
 # read input arguments
 read_arguments "$@"
 
+# temporarily overwrite the log directory
+export ROSWSS_LOG_DIR="${LOG_DIR}"
```
Comment (astumpf) [inline scripts/helper/run_all.sh]: I'm a little bit concerned about possible race conditions here. Just handover the log path to the `run_scripts ` and explicitly set it for the screen call, e.g. `env ROSWSS_LOG_DIR=${MY_PARAM} roswss screen ...`.

See: https://unix.stackexchange.com/questions/495161/import-environment-variables-in-a-bash-script

**Review by astumpf — APPROVED** [review]
Thanks for your work!

**Pull request merged**